doc/cli/operator-sdk_run.md

```diff
@@ -21,6 +21,7 @@ operator-sdk run [flags]
       --install-mode string       [olm only] InstallMode to create OperatorGroup with. Format: InstallModeType=[ns1,ns2[, ...]]
       --include strings           [olm only] Path to Kubernetes resource manifests, ex. Role, Subscription. These supplement or override defaults generated by run/cleanup
       --timeout duration          [olm only] Time to wait for the command to complete before failing (default 2m0s)
+      --olm-namespace string      [olm only] OLM Namespace (default "olm")
```

Comment:
```suggestion
      --olm-namespace string      [olm only] Namespace where the OLM is/will be installed as the OLM operator resources required to be integrated with (default "olm")
```

Reply:
changed it to `Namespace where OLM is installed`